packages/model-registry/upstream/frontend/src/odh/extensions.ts

```diff
@@ -35,9 +35,7 @@ const extensions: (NavExtension | RouteExtension | AreaExtension)[] = [
     flags: {
       required: [SupportedArea.MODEL_CATALOG],
     },
-    featureFlags: ['disableModelCatalog'],
     reliantAreas: [SupportedArea.MODEL_REGISTRY],
```

Comment:
You also don't need `reliantAreas` on either of these because that area is relied on by `SupportedArea.MODEL_CATALOG` already.